.circleci/config.yml

```diff
@@ -315,6 +350,13 @@ workflows:
             tags:
               only: /^v.*$/
 
+      - docker_image_build:
+          filters:
+            branches:
```

Comment:
If slow or we run out of containers, we may run this only on master.

Reply:
It's about 11minutes.

Reply:
So maybe worth running it only on master?

Reply:
I don't mind if we setup proper monitoring on master failure and development policy to fix it with highest priority. But let's do that in future, once we agree on policy and have proper monitoring.